Source/AliveLibAO/Abe.cpp

```diff
@@ -628,7 +632,7 @@ Abe* Abe::ctor_420770(int frameTableOffset, int /*r*/, int /*g*/, int /*b*/)
 
     field_10C_prev_held = 0; // lowest to base class
     field_E6_last_anim_frame = 0;
-    field_110_state = 0;
+    field_110_state = StoneStates::eUnknown_0;
```

Comment:
this isn't just stone states so it should prob only use this enum in the stone state machine func...

Reply:
Maybe rename to something akin to `InternalStates`?